.gitignore

```diff
@@ -50,6 +50,8 @@ examples/ex1[04-9]
 examples/ex1[0-9]p
 examples/ex2[0-9]
 examples/ex2[0-9]p
+examples/ex-normal
+examples/ex-sliding
```

Comment:
I think you can claim Example 28 and Example 29 at this point 😁

Reply:
I made `ex-sliding` into `ex28`, but I am not sure `ex-normal` will end up as an example at all (it uses a mesh not in the main repo and IMO does not demonstrate a capability that's very interesting by itself). It's included right now so reviewers can mess with it.